File: src/test/java/no/nav/fo/veilarbregistrering/FileToJson.java
package no.nav.fo.veilarbregistrering;

import java.nio.file.Files;
import java.nio.file.Paths;

public class FileToJson {

    public static String toJson(String json_file) {
        try {
            byte[] bytes = Files.readAllBytes(Paths.get(FileToJson.class.getResource(json_file).toURI()));
            return new String(bytes);
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
}

File: src/test/kotlin/no/nav/fo/veilarbregistrering/FileToJson.kt
package no.nav.fo.veilarbregistrering

import java.nio.file.Files
import java.nio.file.Paths

object FileToJson {
    fun toJson(json_file: String): String {
        return try {
            val bytes = Files.readAllBytes(Paths.get(FileToJson::class.java.getResource(json_file).toURI()))
            String(bytes)
        } catch (e: Exception) {
            throw RuntimeException(e)
        }
    }
}